### PR TITLE
chore: pre release for 1.1.6

### PR DESCRIPTION
**Pull request by kiragoo**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 ## Release Note 🍻
 
+- Set default resource for containers
```
Comment (Rory-Z) [inline RELEASE.md:3]: This is a new feature? I didn't see a PR for this

File: deploy/charts/emqx-operator/Chart.yaml [hunk 2]
```diff
@@ -15,9 +15,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.4
+version: 1.0.5
```
[the remark this answers is not in the document]
Comment (kiragoo) [reply deploy/charts/emqx-operator/Chart.yaml:18]: ok

File: RELEASE.md (hunk 1 above)
Comment (kiragoo) [reply RELEASE.md:3]: This is not a feature,just changes.

Comment (Rory-Z) [reply RELEASE.md:3]: Plz show code for me

File: RELEASE.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+- Remove the `LifeCycleProbe` for `Telegraf` container
+- Delete `RunAsNonRoot` for default `SecurityContext`
```
[the remark this answers is not in the document]
Comment (kiragoo) [reply RELEASE.md]: done

File: RELEASE.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 ## Release Note 🍻
-
 ### Fixes 🛠
 
-- Fix `.spec.listener.certificate.*.data` not work
\ No newline at end of file
+- Fix Permission Error while deploy `EMQX Cluster` on `EKS`、`ACK` or other platsforms
```
Comment (Rory-Z) [inline RELEASE.md:4]:
```suggestion
- Fix Permission Error while deploy `EMQX Custom Resource ` on `EKS`、`ACK` or other platforms
```

File: RELEASE.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
...
+### Features 🌈
+
+- Add `SecurityContext` for `CR`
```
Comment (Rory-Z) [inline RELEASE.md:7]:
```suggestion
- Add `SecurityContext` for `EMQX Custom Resource`, in order to solve the Permission Denied  when deployed on on `EKS`、`ACK` or other cloud platform
```

**Review by Rory-Z — APPROVED** [review]

**Pull request merged**